tests/plugins/list/blockconversion.html

```diff
@@ -0,0 +1,468 @@
+<!--
+
+Strange output markup for tests:
```

Comment:
Avoid information duplication. You have already provided this info in `blockconversion.js`. 

`blockconversion.js` is where the logic of tests are so it is the best place to put this information. Here it's not needed, as it is only a _resource_ file for the logic.